index.js

```diff
@@ -1,9 +1,9 @@
 function add (a, b) {
-  return a + b
+  return a + b;
 }
 
 function multiply (a, b) {
-  return a * b
+  return a * b;
```

Comment:
That's a good thing, you are following eslint

Reply:
Thanks!